lapis2-docs/astro.config.mjs

```diff
@@ -83,6 +83,27 @@ export default defineConfig({
                         },
                     ]),
                 },
+                {
+                    label: 'Architecture and Dev Docs',
```

Comment:
When I run the container locally, I cant see these links.

Reply:
And also the routes dont work for me

Reply:
Hm, did you start it with the command from the README?
`CONFIG_FILE=../siloLapisTests/testData/testDatabaseConfig.yaml REFERENCE_GENOMES_FILE=../siloLapisTests/testData/reference_genomes.json LAPIS_URL=http://localhost:8080 npm run dev`